iconrpcserver/dispatcher/v3d/version3d.py

```diff
@@ -76,3 +76,13 @@ async def debug_estimateStep(**kwargs):
         score_stub = get_icon_stub_by_channel_name(channel)
         response = await score_stub.async_task().query(request)
         return response_to_json_query(response)
+
+    @staticmethod
+    @methods.add
+    async def debug_getAccount(**kwargs):
+        channel = kwargs['context']['channel']
```

Comment:
need update like [this](https://github.com/icon-project/icon-rpc-server/pull/142/files#diff-2e8b42075f1e5f013ecd3e55847c66dbR74)
```py
async def debug_estimateStep(context, **kwargs):
    channel = context.get('channel')
```